datasources/rpc-transaction-crawler-datasource/src/lib.rs

```diff
@@ -272,13 +272,9 @@ fn signature_fetcher(
                                 let start = Instant::now();
 
                                 if signatures.is_empty() {
-                                    last_fetched_signature = None;
-                                    if most_recent_signature.is_some() {
-                                        until_signature = most_recent_signature;
-                                        most_recent_signature = None;
+                                    if last_fetched_signature.is_none() {
```

Comment:
Not sure what the effect of this is for someone trying to just look back from head.. But this resolves the issue on my end.